src/components/editor/editor-pane/key-map.ts

```diff
@@ -16,6 +16,7 @@ import {
 
 const isVim = (keyMapName?: string) => (keyMapName?.substr(0, 3) === 'vim')
 
+const pass = (_: Editor): void | typeof Pass => undefined
```

Comment:
This function has the same signature as `suppressSave` and is therefore redundant code. Consider merging both functions into one.

Reply:
done